m4/nut_check_libneon.m4

```diff
@@ -17,6 +17,7 @@ if test -z "${nut_have_neon_seen}"; then
 	AC_MSG_CHECKING(for libneon version via pkg-config (0.25.0 minimum required))
 	NEON_VERSION="`pkg-config --silence-errors --modversion neon 2>/dev/null`"
 	if test "$?" != "0" -o -z "${NEON_VERSION}"; then
+		AC_MSG_WARN([could not get pkg-config information for libneon version, using fallback defaults])
```

Comment:
@jimklimov, if possible, don't print anything (non fatal) after an `AC_MSG_CHECKING()`, before its `AC_MSG_RESULT()` has been printed (like here, or below), or the result will end up in a new line with the additional message in-between (which is not exactly the clearest thing to read)